### DISCUSSION TITLE
Assignment submissions and merge conflicts

### DISCUSSION THREAD

**Discussion by NobodyLivesHere** [earthlab-education/Earth-Analytics-2023-01-Intro#323] [! GitHub help] [answered]
So I'm still getting used to the browser github version control for multiple branches.  It seems I cannot merge with the main branch and submit my link for the week 3 analysis because my pull request for last week still hasn't been reviewed and approved.  Is there any way around this?  I imagine other students whose pull requests are still in the queue are going to be running into similar issues.  It also doesn't seem like there's an easy way to check which pull requests we are assigned to review without clicking on each individual pull request. 

When trying to resolve merge conflicts via the terminal in Desktop VSCode I run into the issues of not being able to edit my analysis link in on my branch because the unresolved pull request prevents me from merging.  I know there are ways to force a push but I don't believe student branches would have that capability.  

Below is a screenshot of the error
![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/88915377/93968503-2726-4379-b584-c0d8d429f765)

**Comment by ellenalamont17** [comment]
I asked about the review assignment issue last week. @eculler suggested typing:

is:open is:pr user-review-requested:@me 

into the search bar under the pull request tab. I am missing review assignments, so I have not been able to test if this works yet. But, you might give it a try to find which reviews are assigned to you.

**Comment by eculler** [comment, marked as answer]
@NobodyLivesHere I have merged all the outstanding working PRs from the temperature pages - this should work for you now

**Reply by NobodyLivesHere** [reply]
Thanks a bunch!
